### ISSUE TITLE
debug does not log context

### ISSUE THREAD

**Issue by enesakar** [upstash/rag-chat#43]
```
{
  "timestamp": 1723403597470,
  "logLevel": "INFO",
  "eventType": "RETRIEVE_HISTORY",
  "details": {
    "history": [
      {
        "content": "what is xdf?",
        "role": "user",
        "id": "0"
      }
    ]
  },
  "latency": 89
}
{
  "timestamp": 1723403597578,
  "logLevel": "INFO",
  "eventType": "FORMAT_HISTORY",
  "details": {
    "formattedHistory": "USER MESSAGE: what is xdf?"
  }
}
{
  "timestamp": 1723403597679,
  "logLevel": "INFO",
  "eventType": "FINAL_PROMPT",
  "details": {
    "prompt": "You are a friendly AI assistant augmented with an Upstash Vector Store.\n  To help you answer the questions, a context and/or chat history will be provided.\n  Answer the question at the end using only the information available in the context or chat history, either one is ok.\n\n  -------------\n  Chat history:\n  USER MESSAGE: what is xdf?\n  -------------\n  Context:\n  - xdf is our new product\n- hello\n- hello\n- how are you\n- where is kugaru\n  -------------\n\n  Question: what is xdf?\n  Helpful answer:"
  }
}
{
  "timestamp": 1723403598738,
  "logLevel": "INFO",
  "eventType": "LLM_RESPONSE",
  "details": {
    "response": "XDF is your new product."
  }
}
```

**Comment by ogzhanolguncu** [comment]
This is fixed in https://github.com/upstash/rag-chat/releases/tag/v1.0.6.

**Comment by yunusemreozdemir** [comment]
Closing this issue as it is fixed in PR [#35](https://github.com/upstash/rag-chat/pull/35) and released in [v1.0.6](https://github.com/upstash/rag-chat/releases/tag/v1.0.6).
